crates/client/Cargo.toml

```diff
@@ -23,6 +23,7 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 
 proto = { workspace = true }
+exitfailure = "0.5.1"
```

Comment:
I think we can use `anyhow` to cover this, please checkout out the suggestions for `crates/client/src/api.rs`

Reply:
```suggestion
```